imports/plugins/included/analytics/client/components/settings.js

```diff
@@ -0,0 +1,86 @@
+import React, { Component } from "react";
+import PropTypes from "prop-types";
+import _ from "lodash";
+import {
+  CardGroup,
+  SettingsCard,
+  Form
+} from "/imports/plugins/core/ui/client/components";
+import { ReactionAnalyticsPackageConfig } from "/lib/collections/schemas/analytics";
+
+const analyticsProviders = [
```

Comment:
This should probably be declared in the `settings` portion of `registry.js` rather than in a presentation component. Then you can just grab it when you grab the rest of the settings.